client/extensions/woocommerce/app/products/product-create.js

```diff
@@ -66,8 +68,24 @@ class ProductCreate extends React.Component {
 	}
 
 	onSave = () => {
-		const { siteId, product } = this.props;
-		this.props.createProduct( siteId, product );
+		const { siteId, product, translate } = this.props;
+
+		const successAction = successNotice(
+			translate( '%(product)s successfully created.', {
```

Comment:
:information_source: This string doesn't exist elsewhere without a **context** (or with a different one). You can probably drop the context. If needed, you can use a translator comment instead to clarify meaning.